packages-common/pluggable-widgets-tools/test-config/wdio.conf.js

```diff
@@ -1,26 +1,41 @@
-const path = require("path");
-const fs = require("fs");
+const { join } = require("path");
+const { existsSync, mkdirSync } = require("fs");
 const debug = process.env.DEBUG;
+const browser = process.env.BROWSER || "chrome";
 const url = process.env.URL || "https://localhost:8080/";
 
-const e2ePath = "../../../dist/e2e/";
-if (!fs.existsSync(e2ePath)) {
-    fs.mkdirSync(e2ePath, { recursive: true });
+const e2ePath = join(process.cwd(), "dist/e2e/");
+if (!existsSync(e2ePath)) {
+    mkdirSync(e2ePath, { recursive: true });
 }
 
+console.warn("Starting wdio with ", url, browser);
+
 exports.config = {
     before() {
-        require('@babel/register');
-        require("ts-node").register({ files: true, project: "../../../tests/e2e/tsconfig.json" });
+        require("@babel/register");
+        require("ts-node").register({ files: true, project: join(process.cwd(), "./tests/e2e/tsconfig.json") });
     },
     host: "127.0.0.1",
     port: 4444,
-    specs: [ "../../../tests/e2e/**/*.spec.js","../../../tests/e2e/**/*.spec.ts" ],
+    specs: [join(process.cwd(), "./tests/e2e/**/*.spec.js"), join(process.cwd(), "./tests/e2e/**/*.spec.ts")],
     maxInstances: debug ? 1 : 5,
-    capabilities: [ {
-        maxInstances: debug ? 1 : 5,
-        browserName: "chrome"
-    } ],
+    capabilities: [
```

Comment:
🎄 wouldn't it be easier to just
```suggestion
    capabilities: [{
        browserName: browser,
        "goog:chromeOptions": {
             args: debug ? ["--no-sandbox"] : ["--no-sandbox", "--disable-gpu", "--disable-extensions"]
         }
     }],
```

Reply:
Done